mongo/integration/unified_spec_test.go

```diff
@@ -821,13 +821,11 @@ func verifyTestOutcome(mt *mtest.T, outcomeColl *outcomeCollection) {
 	if outcomeColl.Name != "" {
 		collName = outcomeColl.Name
 	}
-	coll := mt.GlobalClient().Database(mt.DB.Name()).Collection(collName)
+	coll := mt.GlobalClient().Database(mt.DB.Name()).Collection(collName, checkOutcomeOpts)
```

Comment:
Minor change: The `CollectionOptions` can be passed directly to the `Collection` function rather than calling `Clone` below.